test/unit/pipeline/show-nonprinting.ts

```diff
@@ -0,0 +1,160 @@
+/*
+	File: test/unit/pipeline/show-nonprinting.js
+	cpuabuse.com
+*/
+
+/**
+ * Tests `showNonprinting`.
+ */
+
+import { strictEqual } from "assert";
+import {
+	tabCharacter as inputTabCharacter,
+	lineFeedCharacter as inputlineFeedCharacter,
+	carriageReturnCharacter as inputCarriageReturnCharacter,
+	space as inputSpace,
+	specialCharacters as inputSpecialCharacters,
+	numbers as inputNumbers,
+	letters as inputLetters,
+	del as inputDel,
+	getExtendedLower as getInputExtendedLower,
+	extendedHigher as inputExtendedHigher,
+	unicodeCharacter as inputUnicodeCharacter
+} from "../../input/pipeline";
+import {
+	tabCharacter as expectedTabCharacter,
+	lineFeedCharacter as expectedLineFeedCharacter,
+	carriageReturnCharacter as expectedCarriageReturnCharacter,
+	space as expectedSpace,
+	specialCharactersCount as expectedSpecialCharactersCount,
+	specialCharacters as expectedSpecialCharacters,
+	numbers as expectedNumbers,
+	numbersCount as expectedNumbersCount,
+	letters as expectedLetters,
+	lettersCount as expectedLettersCount,
+	del as expectedDel,
+	extendedCharacters as expectedExtendedCharacters,
+	extendedHigher as expectedExtendedHigher,
+	unicodeCharacter as expectedUnicodeCharacter
+} from "../../expected/pipeline";
+import { showNonPrinting } from "../../../src/pipeline/show-nonprinting";
+
+export function testShowNonPrinting() {
+	describe("non-printing", function() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['describe' is not defined.](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5042286)